packages/core/admin/admin/src/components/GuidedTour/utils/persistStateToLocaleStorage.js

```diff
@@ -26,6 +27,9 @@ const persistStateToLocaleStorage = {
   setSkipped: value => {
     localStorage.setItem(SKIPPED, stringify(value));
   },
+  setGuidedTourVisible: value => {
+    localStorage.setItem(VISIBLE, stringify(value));
```

Comment:
can it sometimes be set to `false` ?

Reply:
it will, when the guided tour is completed, will add it in my next commit!

Reply:
no actually you are right, it will only be `setGuidedTourVisible(true)`
once the guided tour is over we will have something like `removeGuidedTourVisibile` from locale storage, WDYT?